core/store/migrate/migrations/0100_bootstrap_config.sql

```diff
@@ -0,0 +1,117 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE bootstrap_contract_configs
+(
+    bootstrap_spec_id       INTEGER PRIMARY KEY,
+    config_digest           bytea                    NOT NULL,
+    config_count            bigint                   NOT NULL,
+    signers                 bytea[],
+    transmitters            text[],
```

Comment:
These are allowed to be null?

Reply:
I've used the same format as the current ocr2 contract_configs as the same code will populate them

Reply:
having them as not null does imply an assumption about the implementation of the onchain contracts, however terra, solana and evm contracts all have those checks so we can add a NOT NULL to start